crates/swc_ecma_parser/tests/span/js/decl/class-no-close-brace.js

```diff
@@ -0,0 +1,3 @@
+class Test {
+
+  method() {}
```

Comment:
Waiting on #4186 in order to update the .js.spans below.

Reply:
Thank you!

Actually, I tried error recovery several time and failed, but I think I can do it if I try in some gradual way...
The problem was an infinite loop in some test cases

Reply:
This is resolved now.

I think what would be good for error handling is to look at what TypeScript does for certain things and work incrementally on it. TypeScript seems to work well in the editor.